README.md

```diff
@@ -298,19 +298,25 @@ Below are listed all the variables you can customize (you ma also want to take a
   vars:
     # optional openwisp2 modules
     openwisp2_network_topology: false
-    # you may set one of these variables to true if you need to
-    # use the development version of a specific openwisp2 module
-    openwisp2_controller_dev: false
-    openwisp2_users_dev: false
-    openwisp2_django_netjsonconfig_dev: false
-    openwisp2_django_x509_dev: false
-    openwisp2_netjsonconfig_dev: false
-    openwisp2_network_topology_dev: false
-    openwisp2_django_netjsongraph_dev: false
+    # you may replace the values of these variables with any URL
```

Comment:
these line changes should not show here (until 312), do you have a clue of why this is happening?

Reply:
I think it's because my README on the branch wasn't the same as the one in openwip:master. Is there a way to fix it? Do I merge openwisp:master to the branch and then write the fix?